components/shared/embedder/input_events.rs

```diff
@@ -147,6 +147,7 @@ pub struct TouchEvent {
     pub event_type: TouchEventType,
     pub id: TouchId,
     pub point: DevicePoint,
+    pub cancelable: bool,
```

Comment:
I think this field should be private for now. We can set cancelable to false via a `#[doc(hidden)]` pub method (like we did for the sequence id).